version

```diff
@@ -1 +1 @@
-0.0.56
+0.0.57

```

Comment:
This is a major change, should we bump the version segments?

Reply:
I don't think we need to do it now as we will upgrade it to 1.0.0 this sprint